reconcile/external_resources/integration.py

```diff
@@ -41,12 +47,32 @@ def fetch_current_state(
         ri.add_current(cluster, namespace, "Job", r.name, r)
 
 
+def get_aws_api(
+    query_func: Callable,
+    account_name: str,
+    region: str,
+    secret_reader: SecretReaderBase,
+) -> AWSApi:
+    account = [
+        acc
+        for acc in aws_accounts_query(query_func).accounts or []
+        if acc.name == account_name
+    ][0]
```

Comment:
since this is a new gql query always find by account name, we can use server side filter to pass account name https://github.com/app-sre/qontract-server?tab=readme-ov-file#field-equality-predicate

Reply:
Added. Thanks for the suggestion. PTAL